frontend/src/shared/components/tasque-dropdown/dropdown.component.sass

```diff
@@ -0,0 +1,85 @@
+@import 'src/colors.sass'
+    
+.dropdown_wrpr
+    font-family: Nunito
+    font-style: normal
+    color: colorscheme(black-100)
+    display: flex
+    flex-direction: column
+    font-size: 14px
+    position: relative
+   
+.options_container
+    margin-top: 2px
+    background-color: colorscheme(white-100)
+    border: 2px solid colorscheme(gray-300)
+    border-radius: 10px
+    max-height: 140px
+    overflow: auto
+    width: 99.5%
```

Comment:
is there a reason it's not 100%?

Reply:
It seems a little bigger than a dropdown button with 100%, probably because of the borders